### PR TITLE
SEO-175680-Xamarin-UG-page-with-redirects-links

### PR DESCRIPTION
**Pull request by Edithabich**
Hi @MallikaRK ,
Have made changes on the redirect links. Please review.
Thanks.

Regards,
Edith.

### CODE REVIEW

**Comment by Edberg-Vasan** [comment]
rebuild

File: Xamarin/Numeric-Entry/Getting-Started.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This section explains you the steps to configure a SfNumericTextBox control in
 
 ## Assembly deployment
 
-After installing [Essential Studio for Xamarin](https://www.syncfusion.com/downloads/xamarin), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
+After installing [Essential Studio for Xamarin](https://www.syncfusion.com/account/manage-trials/downloads), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
```
Comment (Yvone-Atieno) [inline Xamarin/Numeric-Entry/Getting-Started.md:16]: Revert changes

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Numeric-Entry/Getting-Started.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ You can add SfNumericTextBox reference using one of the following methods:
 
 **Method 1: Adding SfNumericTextBox reference from nuget.org**
 
-Syncfusion Xamarin components are available in [nuget.org](https://www.nuget.org/). To add SfNumericTextBox to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Xamarin.SfNumericTextBox](https://www.nuget.org/packages/Syncfusion.Xamarin.SfNumericTextBox), and then install it.
+Syncfusion Xamarin components are available in [nuget.org](https://www.nuget.org). To add SfNumericTextBox to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Xamarin.SfNumericTextBox](https://www.nuget.org/packages/Syncfusion.Xamarin.SfNumericTextBox), and then install it.
```
Comment (Yvone-Atieno) [inline Xamarin/Numeric-Entry/Getting-Started.md:28]: revert this change

File: Xamarin/Shimmer/Getting-Started.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ This section explains the steps required to configure the shimmer.
 
 ## Assembly deployment
 
-After installing [Essential Studio for Xamarin](https://www.syncfusion.com/downloads/xamarin), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
+After installing [Essential Studio for Xamarin](https://www.syncfusion.com/account/manage-trials/downloads), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\Xamarin\lib.
```
Comment (Yvone-Atieno) [inline Xamarin/Shimmer/Getting-Started.md:16]: Revert this change

File: Xamarin/Shimmer/Getting-Started.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ You can add SfShimmer reference using one of the following methods:
 
 **Method 1: Adding SfShimmer reference from nuget.org**
 
-Syncfusion Xamarin components are available in [nuget.org](https://www.nuget.org/). To add SfShimmer to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Xamarin.Core](https://www.nuget.org/packages/Syncfusion.Xamarin.Core), and then install it.
+Syncfusion Xamarin components are available in [nuget.org](https://www.nuget.org). To add SfShimmer to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Xamarin.Core](https://www.nuget.org/packages/Syncfusion.Xamarin.Core), and then install it.
```
Comment (Yvone-Atieno) [inline Xamarin/Shimmer/Getting-Started.md:28]: Revert this

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: Xamarin/Numeric-Entry/Getting-Started.md (hunk 2 above)
Comment (Edithabich) [reply Xamarin/Numeric-Entry/Getting-Started.md:28]: resolved

File: Xamarin/Shimmer/Getting-Started.md (hunk 3 above)
Comment (Edithabich) [reply Xamarin/Shimmer/Getting-Started.md:16]: resolved

File: Xamarin/Shimmer/Getting-Started.md (hunk 4 above)
Comment (Edithabich) [reply Xamarin/Shimmer/Getting-Started.md:28]: resolved

File: Xamarin/Installation/mac-installer/how-to-download.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ You should initiate an evaluation if you have already obtained our components th
 
 ## Download the License Version
 
-1. Syncfusion licensed products will be available in the [License & Downloads](https://www.syncfusion.com/account/downloads) page under your registered Syncfusion account.
+1. Syncfusion licensed products will be available in the [License & Downloads](https://www.syncfusion.com/account/login) page under your registered Syncfusion account.
```
Comment (Yvone-Atieno) [inline Xamarin/Installation/mac-installer/how-to-download.md:66]: Check this link

Comment (Edithabich) [reply Xamarin/Installation/mac-installer/how-to-download.md:66]: fixed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**